### PR TITLE
Create event page base UI

### PR DESCRIPTION
**Pull request by WenjunYuAnny**

### CODE REVIEW

File: events/static/events/images/dot-grid.webp
Comment (Dhiren-NYU) [inline events/static/events/images/dot-grid.webp:1]: Where have we used this webp?

File: events/static/events/images/memphis-mini.webp
Comment (Dhiren-NYU) [inline events/static/events/images/memphis-mini.webp:1]: same cannot find reference to this file. Looks unused

File: events/static/events/images/stripes-light.webp
Comment (Dhiren-NYU) [inline events/static/events/images/stripes-light.webp:1]: same

File: events/static/events/images/united-squares-negative.webp
Comment (Dhiren-NYU) [inline events/static/events/images/united-squares-negative.webp:1]: same

File: events/templates/events/base.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{% load static %}
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>{% block title %}Default Title{% endblock %}</title>
+    <!-- Bootstrap CSS -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-T3c6CoIi6uLrA9TneNEoa7RxnatzjcDSCmG1MXxSR1GAsXEV/Dwwykc2MPK8M2HN" crossorigin="anonymous">
```
[the remark this answers is not in the document]
Comment (WenjunYuAnny) [reply events/templates/events/base.html:9]: Sure I can do that!

File: events/templates/events/create-event.html [hunk 2]
```diff
@@ -1,29 +1,146 @@
...
-    <input type="text" name="event_name" id="event_name" placeholder="Add Event Name" required>
+{% extends "events/base.html" %}
+
+{% block title %}
+Create Event
+{% endblock %}
+
+{% block extra_styles %}
+<style>
+    .sticky-note-container {
```
[the remark this answers is not in the document]
Comment (WenjunYuAnny) [reply events/templates/events/create-event.html:9]: I have to say I'm not sure for now how frequent this class will be used on other pages so for now I keep it here. If in the future, it's a frequent element, I can move it to the base template.

File: events/templates/events/create-event.html [hunk 3]
```diff
@@ -1,29 +1,146 @@
...
+    width: 500px;  /* Desktop size */
```
[the remark this answers is not in the document]
Comment (WenjunYuAnny) [reply events/templates/events/create-event.html:10]: Below I have a line for smaller screen and tested it. Right now it looks like this on the smaller screen. I can adjust it more if needed.
image

File: events/static/events/images/dot-grid.webp
Comment (WenjunYuAnny) [reply events/static/events/images/dot-grid.webp:1]: Will delete this one. This is a leftover for try outs.

File: events/static/events/images/memphis-mini.webp
Comment (WenjunYuAnny) [reply events/static/events/images/memphis-mini.webp:1]: Same as previous one.

File: events/static/events/images/stripes-light.webp
Comment (WenjunYuAnny) [reply events/static/events/images/stripes-light.webp:1]: Same as previous one.

File: events/static/events/images/united-squares-negative.webp
Comment (WenjunYuAnny) [reply events/static/events/images/united-squares-negative.webp:1]: Same as previous one.

**Review by Dhiren-NYU — APPROVED** [review]

**Pull request merged**